docs/quickstart.rst

```diff
@@ -77,3 +78,5 @@ On other systems:
     $ python3 example_bot.py
 
 Now you can try playing around with your basic bot.
+
+You can find more examples in the `examples directory <https://github.com/nextcord/nextcord/blob/stable/examples/>`_ on GitHub.
```

Comment:
the stable branch is an old attempt at a stable version, maybe change to master for now